### DISCUSSION TITLE
Button maker in editor tool.

### DISCUSSION THREAD

**Discussion by Giantbean** [google/model-viewer#4866] [Ideas]
This may be out of scope but the editor tool https://modelviewer.dev/editor/ would be greatly enhanced if it had tools to make buttons like the Web XR example https://modelviewer.dev/examples/augmentedreality/#webXR

**Comment by elalish** [comment]
Yeah, those buttons are pretty generic and all they do is set the `src`. I would hope other website editors would already have better functionality there. We just focus on things that are 3D-specific, rather than general 2D web design.

**Reply by Giantbean** [reply]
Good point.
